data_processing/notebooks/model_conversion.ipynb

```diff
@@ -0,0 +1,1022 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "converter = tf.lite.TFLiteConverter.from_saved_model(f\"{model_path}_tf\")\n",
+    "converter.optimizations = [tf.lite.Optimize.DEFAULT]\n",
+    "converter.target_spec.supported_types = [tf.float16]\n",
+    "tflite_model_fp16 = converter.convert()\n",
+    "\n",
+    "# Save the model.\n",
+    "with open(f\"{model_path}_fp16.tflite\", 'wb') as f:\n",
+    "    f.write(tflite_model_fp16)\n",
+    "    \n",
+    "inference_tflite(f\"{model_path}_fp16.tflite\", frame_for_model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "73f0ed6d",
+   "metadata": {},
+   "source": [
+    "## INT8"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "b844dd84",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "def representative_dataset():\n",
+    "    df = pd.read_csv('../data_processing/example_data/one_person_close.csv', delimiter=',', skiprows=[0], index_col=0)\n",
```

Comment:
How about taking more representative data, e.g. one of training dataset files? And changing row temperatures to normalized values (because the model uses normalized values as input)